src/RollbarSharp.Tests/Serialization/TraceModelTest.cs

```diff
@@ -49,11 +51,32 @@ public void when_creating_from_thrown_exception_should_have_Exception_model()
             }
         }
 
+        [Test()]
+        public void when_creating_from_thrown_within_reflection_call_exception_should_have_Exception_model()
+        {
+            var method = this.GetType().GetMethod("MethodWithException");
+            try
+            {
+                method.Invoke(this, new object[]{ null });
+                Assert.Fail("La llamada anterior debe lanzar una excepción");
```

Comment:
This will fail in our build server, the main culture is english, you can force the NUnit to run in a given culture, fix that